data/json/items/tool_armor.json

```diff
@@ -1213,6 +1211,27 @@
         "coverage" : 0,
         "material_thickness" : 0
     },
+    {
+        "id" : "fc_hairpin",
+        "type" : "TOOL_ARMOR",
+        "category" : "clothing",
+        "symbol" : "[",
+        "color" : "pink",
+        "name" : "fancy hairpin",
+        "description" : "A pink hairpin with cute flower textures.",
+        "price" : 1000,
+        "material" : ["plastic"],
```

Comment:
That is why it is only level 1. (All other tools are 3 or higher).

And well, it just being hard enough is it. iirc, I used a toothpick once as a lockpick. (I was surprised it worked).